### PR TITLE
Feature UI audio convert

### PR DESCRIPTION
**Pull request by SaulCM**
Added Http request class, parameters model and text extractor view.

### CODE REVIEW

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    /**
+     * Audio Request Form stores parameters for an audio request
+     */
+    public AudioRequestForm() {
+        
+    }
+
+    /**
+     * Add filepath parameter
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java]: the comment should start in the third person

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+     * @param filepathValue
+     */
+    public void addFilepath(String filepathValue) {
+        addParameters(new Parameter(FILE, filepathValue, true));
+    }
+
+    /**
+     * Add format parameter
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java]: the comment should start in the third person

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+     * @param formatValue
+     */
+    public void addFormat(String formatValue) {
+        addParameters(new Parameter(FORMAT, formatValue, false));
+    }
+
+    /**
+     * Add bitrate parameter
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java]: the comment should start in the third person

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    /**
+     * Add metadata parameter
+     * @param metadataValue
+     */
+    public void addMetadata(String metadataValue) {
+        addParameters(new Parameter(HZ, metadataValue, false));
+    }
+
+    /**
+     * @return bodyParameters
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java:97]: Is missing Description of the method (javaDoc)

File: src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.entity.mime.MultipartEntityBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.fundacion.jala.converter.view.Models.IrequestForm;
+import org.fundacion.jala.converter.view.Models.Parameter;
+import java.io.*;
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java]: It should be the specific library and not a general *.

File: src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+        FileOutputStream outputStream = new FileOutputStream(new File(filePath));
+        int inByte;
+        while((inByte = inputStream.read()) != -1)
+            outputStream.write(inByte);
+        inputStream.close();
+        outputStream.close();
+    }
+    /**
+     * Retrieves a  token from the endpoint from a username and password.
+     * @return
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java]: All methods should provide a description of their params and returns on their javadocs, not only the name.

File: src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+ * @version 1.0
+ */
+package org.fundacion.jala.converter.view.Models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AudioRequestForm implements IrequestForm{
+    public List<Parameter> bodyParameters = new ArrayList<>();
+    public final String URL = "http://localhost:8080/api/extractAudio";
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/view/Models/AudioRequestForm.java]: Change extractAudio to convertAudio

File: src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java [hunk 8]
```diff
@@ -0,0 +1,158 @@
+/**
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/view/Controller/ClientRequest.java]: package is lowerCase Controller

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**